requirements.txt

```diff
@@ -4,3 +4,4 @@ pandas>=1.2.0
 scikit-learn>=0.21.2
 sqlalchemy>=1.3.6
 pytest>=5.0.1
+cytominer-database==0.3.4
```

Comment:
It makes me a bit nervous to include `cytominer-database` as a required package. I'll look this up concurrently as well, but do you know if there is a way to specify optional parameters?

It would be nice to use pycytominer without this strict requirement, and if one would like to use pycytominer in conjunction, they would have to specify.